### PR TITLE
Align templates to add new standard parameters - Part 2

### PR DESCRIPTION
**Pull request by sspeiche**
@rcernich PTAL

I will update the webserver/tomcat icon references in another PR

### CODE REVIEW

**Comment by rcernich** [comment]
Why are we adding "hidden" tags to the templates?

**Comment by sspeiche** [comment]
@rcernich In discussions with Kevin and the regular group, we weighed the pros and cons of various approaches to handling the larger number of templates. Instead of deleting "less important ones" we concluded it would be best to have them but have a way to filter out these "less important ones". Since this hidden concept exists for image-streams, which can be used to "hide" deprecated image-stream tags, it was agreed we could tag as hidden. The UI could filter these out. They are still there for usage.

So I was given the action to selectively mark those hidden which didn't seem to be the most used (20% case) and focused more on leaving higher value templates the focus (shown). I spent a fair amount of time on this to get to a good sweet spot. Let me know if some of these we should adjust.

We have follow up actions to fix this better longer term, this is just a step for 3.7 with the new UX and template service broker / catalog.

File: datagrid/datagrid71-mysql.json [hunk 1]
```diff
@@ -3,11 +3,15 @@
...
     "metadata": {
         "annotations": {
-            "iconClass": "icon-jboss",
-            "description": "Application template for JDG 7.1 and MySQL applications.",
-            "tags": "datagrid,jboss,xpaas",
+            "iconClass": "icon-datagrid",
+            "tags": "datagrid,jboss",
             "version": "1.4.6",
-            "openshift.io/display-name": "Red Hat JBoss Data Grid 7.1 + MySQL (Ephemeral with https)"
+            "openshift.io/display-name": "Boss Data Grid 7.1 + MySQL (Ephemeral with https)",
```
Comment (spadgett) [inline datagrid/datagrid71-mysql.json]: s/Boss/JBoss -- missing the `J`

Comment (sspeiche) [reply datagrid/datagrid71-mysql.json]: ack, thanks...fixed

**Comment by sspeiche** [comment]
@rcernich Let me know if there are any other updates needed. This follows in what was done/merged for Part 1...just for all the other templates per discussion.

**Pull request merged**